### ISSUE TITLE
Description - wrong key-combination

### ISSUE THREAD

**Issue by leomayer** [morassman/atom-commander#67]
For opening a panel you write in the summary CTRL-T
On my Linux-system this is CTRL-t 

Same applies to remove a panel: CTRL-R ==> CTRL-r

**Comment by morassman** [comment]
Strange... This doesn't seem to be a problem on OSX and Windows. I'll make change it to not be case-sensitive.

What distribution of Linux are you using?

**Comment by leomayer** [comment]
OpenSuSe 42.1
But could be as well due to other installed packages - don't know. Cuz when I look on the Keybindings and search for CTRL-R it will list only CTRL-r and there I have two bindings - one for Atom Commander and one for Symbol View with Selector (.platform-linux atom-text-editor) and Command (symbols-view:toggle-file-symbols).
Key-mapping-conflict?

**Comment by morassman** [comment]
It may be a key-mapping conflict. I'll look into it.

**Comment by morassman** [comment]
In atom-commander's keymaps file the shortcuts are specified as:

```
'ctrl-t': 'atom-commander:add-tab'
'ctrl-r': 'atom-commander:remove-tab'
```

As far as I know this should work with either 't' or 'T'. 

I tried this myself and things worked as I expected. When the editor has focus and I press ctrl-r then the symbols view's command is triggered. When atom-commander has focus then its remove tab command is triggered.

Do you have this problem even when atom-commander has focus?

**Comment by leomayer** [comment]
Hmmm... retesting your last comment and the whole thing turns out in a strange way.

First of all - no probs when atom-commander has the focus ... seems like it was some kind of focusing issue :-/

BUT the strange things start to happen when I have the focus in the 'address bar'. CTRL-t worx but when I'm on the right side => a tab is added on the left side ! (instead of the right)
CTRL-r does NOT work (neither left nor right) when focus is in the address bar.

**Comment by morassman** [comment]
Aaah. I know what's happening.

**Ctrl-t**
Ctrl-t add a tab to the panel (left/right) that has focus, but the focus doesn't change when one select the address bar. I'll have to fix this.

You can confirm it like so:
- Select the right panel
- Select the right address bar
- Press Ctrl-t
- The tab will appear on the right

If you select the left panel first and then the right address bar, then the tab will be on the left.

**Ctrl-r**
I looked into this and I _think_ it can be solved by associating the shortcut key with a nested selector. The Ctrl-r is bound to the text editor where you edit files, but the address bar uses the same instance of such a text editor, just with a single line. So ctrl-r ends up triggering the command associated with the text editor and not atom-commander. I'll look into this as well.

**Comment by leomayer** [comment]
Reg. Ctrl-t: I can confirm that behaviour. When I follow your steps ==> it worx as you describe 👍

**Comment by morassman** [comment]
This should now be fixed as part of v0.9.0
